cpp/src/arrow/chunked_array_test.cc

```diff
@@ -204,9 +210,9 @@ TEST_F(TestChunkedArray, Validate) {
   Construct();
   ASSERT_OK(one_->ValidateFull());
 
+  // Invalid if different chunk types
   arrays_one_.push_back(gen.String(50, 0, 10, 0.1));
-  Construct();
-  ASSERT_RAISES(Invalid, one_->ValidateFull());
```

Comment:
I think the previous test can be restored right?

Reply:
Yes, and it will pass, but this test is incorrect because it constructs a ChunkedArray with chunks of different types (Int32 and String).

Reply:
Ah, I thought the intent was to ensure that ChunkedArray::ValidateFull would catch the type mismatch.

Reply:
Nevermind, restoring test.